deploy.sh

```diff
@@ -26,13 +31,33 @@ function iam_mapping {
     helm template aws ./helm --set aws.role.node="${node}",aws.role.user="${user}" | kubectl apply -n kube-system -f -
 }
 
+function get_kubeconfig {
+    eksctl utils write-kubeconfig hindsight-kubernetes-${ENVIRONMENT_NAME}
+}
+
+function check_dependencies {
+    type eksctl 1> /dev/null
+    type openssl 1> /dev/null
+    type helm 1> /dev/null
+    type jq 1> /dev/null
+}
```

Comment:
If one of these does not exist we will return:

./deploy.sh: line 42: type: jq: not found

Reply:
And exit

Reply:
👍